### PR TITLE
Manual: split and redo alignment representation

### PR DESCRIPTION
**Pull request by marehr**
* Alignment Representation: New topic overview
* Split Align Object and Alignment Graph; New subpages

### CODE REVIEW

**Comment by xenigmax** [comment]
Maybe you have to merge the current manual branch to your split_alignment branch.
http://cdash.seqan.de/testDetails.php?test=31314869&build=170249

**Comment by marehr** [comment]
Oh, I reused the images from the built-directory of the docs. I changed that to use the source-directory. We will see if the build will finish :v:

File: manual/source/Tutorial/DataStructures/Alignment/AlignmentGaps.rst [hunk 1]
```diff
@@ -2,36 +2,34 @@
...
-  This tutorial introduces you to the two data structures that can be used to represent an alignment in SeqAn.
+  This tutorial introduces you to the gaps data structures that can be used to represent an alignment in SeqAn.
   You will learn basic techniques to create and modify such data structures and how to access certain information from these data structures.
 
 Difficulty
   Basic
 
 Duration
-  0:45h
+  0:15h
```
Comment (xenigmax) [inline manual/source/Tutorial/DataStructures/Alignment/AlignmentGaps.rst]: "15 min"  I think

File: manual/source/Tutorial/DataStructures/Alignment/AlignmentGraph.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Learning Objective
+  This tutorial introduces you to the graph data structures that can be used to represent an alignment in SeqAn.
+  You will learn basic techniques to create and modify such data structures and how to access certain information from these data structures.
+
+Difficulty
+  Basic
+
+Duration
+  0:15h
```
Comment (xenigmax) [inline manual/source/Tutorial/DataStructures/Alignment/AlignmentGraph.rst]: 15 min?

File: manual/source/Tutorial/DataStructures/Alignment/AlignmentGraph.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+Before adding nodes to the graph ``align`` prints the empty adjacency and edge
+list.
+
+.. includefrags:: demos/tutorial/alignment/graph.cpp.stdout
+   :fragment: output_init
+
+Before we construct the alignment we print the unmodified AlignmentGraph.
+Then we add some alignment information to the graph.
+In order to add an ungapped alignment segment we have to add an edge connecting two nodes of different input sequences.
+To do so we can use the function :dox:`Graph#addEdge` and specify the two vertices that should be connected.
```
Comment (xenigmax) [inline manual/source/Tutorial/DataStructures/Alignment/AlignmentGraph.rst]: I think it's better to use one notation - "node" or "vertex" .

**Comment by xenigmax** [comment]
LGTM !

**Comment by marehr** [comment]
only bam_io failed...

**Comment by xenigmax** [comment]
I rerun the test and it works. LGTM

**Pull request merged**